app/models/topic_link_click.rb

```diff
@@ -70,17 +70,17 @@ def self.create_from(args = {})
       end
     end
 
-    link = TopicLink.select([:id, :user_id])
-
     # test for all possible URLs
-    link = link.where(Array.new(urls.count, "url = ?").join(" OR "), *urls)
+    link = TopicLink.select([:id, :user_id]).where(url: urls)
```

Comment:
I prefer `pluck` to `select` because with select you get an object back that LOOKS like your model (TopicLink) but does not contain all the fields to make it work. `pluck` will only return an array which means you can't accidentally think you have more data than you have.

Reply:
I too prefer to use pluck, but in this case a TopicLink object is used later and carrying each attribute in a different variable makes everything more complex. I can still make the change if you want.

Reply:
If that is the case I would prefer us to not `select` and include all fields. 

It's the object that "looks like" a link but is not a full link that I believe leads to bugs down the road.

Reply:
Thanks for the suggestion, I made the changes.